src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -133,6 +133,7 @@ type ComparisonSideProps = {
   compareButtonMode: CompareButtonMode;
   dragMode: DragMode;
   loadingTime: number;
+  leftSideLoaded?: boolean;
```

Comment:
This component shouldn't know about "left-right" logic. Use some abstract naming: "loadTileset: boolean"